packages/native/src/lib/ElementAssertion.ts

```diff
@@ -68,6 +66,33 @@ export class ElementAssertion extends Assertion<ReactTestInstance> {
     });
   }
 
+  /**
+   * Check if the element is empty.
+   *
+   * @example
+   * ```
+   * expect(element).toBeEmpty();
+   * ```
+   *
+   * @returns the assertion instance
+   */
+  public toBeEmpty(): this {
+    const error = new AssertionError({
+      actual: this.actual,
+      message: `Expected element ${this.toString()} to be empty.`,
+    });
+    const invertedError = new AssertionError({
+      actual: this.actual,
+      message: `Expected element ${this.toString()} NOT to be empty.`,
+    });
+
+    return this.execute({
+      assertWhen: isEmpty(this.actual.children),
```

Comment:
Maybe we can replace this helper function with React's `Children.count(..)` function, wdyt?
```suggestion
      assertWhen: Children.count(this.actual.children) === 0,
```

Reply:
Sure. Thanks Jose! 🚀